setup.py

```diff
@@ -552,25 +551,25 @@ def copy_extensions_to_source(self):
     install_requires=AFFIRM_REQUIRED,
     # https://stackoverflow.com/questions/28509965/setuptools-development-requirements
     # Install dev requirements with: pip install -e .[dev]
-    # extras_require={
-    #     "dev": DEV_REQUIRED,
-    #     "ci": CI_REQUIRED,
-    #     "gcp": GCP_REQUIRED,
-    #     "aws": AWS_REQUIRED,
-    #     "bytewax": BYTEWAX_REQUIRED,
-    #     "redis": REDIS_REQUIRED,
-    #     "snowflake": SNOWFLAKE_REQUIRED,
-    #     "spark": SPARK_REQUIRED,
-    #     "trino": TRINO_REQUIRED,
-    #     "postgres": POSTGRES_REQUIRED,
-    #     "azure": AZURE_REQUIRED,
-    #     "mysql": MYSQL_REQUIRED,
-    #     "ge": GE_REQUIRED,
-    #     "hbase": HBASE_REQUIRED,
-    #     "go": GO_REQUIRED,
-    #     "docs": DOCS_REQUIRED,
-    #     "cassandra": CASSANDRA_REQUIRED,
-    # },
+    extras_require={
```

Comment:
**Ross**: Was this change made by accident?

Reply:
No but it's unrelated. These were commented out because:
```
"google-cloud-datastore>=2.1.*,<3",
```
was causing errors but
```
"google-cloud-datastore>=2.1.0,<3",
```
doesn't so I fixed it and now `pip install -e .[dev]` actually works

Reply:
😍 😍 😍